apps/heureka-next/src/components/Services/ServicesList/ServiceListItem/index.ts

```diff
@@ -0,0 +1,6 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and Greenhouse contributors
```

Comment:
Did you add the headers manually? This might lied to a conflict... 
- Not sure if it is ok hosting the app in Juno having different header as the oder apps
- If you forget in one file to add it manually it will then inject the one set by the github action:
https://github.com/cloudoperators/juno/blob/main/.github/licenserc.yaml#L5